CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.7.1] - 2023-07-20
+
+### Added
+
+- x509pop node attestor emits a new selector with the leaf certificate serial number (#4216)
+- HTTPS server in the OIDC Discovery Provider can now be configured to use a certificate file (#4190)
+- Option to log source information in server and agent logs (#4246)
+
+### Changed
+
+- Improvement in entry lookup logic when fetching JWT-SVIDs (#4211)
```

Comment:
I dunno if this is worth mentioning since it is more or less just a code maintenance improvement?

Reply:
If it translates into a meaningful perf benefit, we could say something like "reduced CPU/memory usage when fetching JWT-SVIDs"

Reply:
It will probably not be a noticeable perf improvement in most cases. I'm now more inclined to remove this.